Automattic/vip-cli
# Don't pipe output through less

This was originally meant to be useful but it's actually more frustrating and a bit of an anti-pattern when working with a CLI tool. It's especially frustrating because you can't easily disable the behavior either. Plus, we're assuming that less or PAGER will be available which may not always be the case.

Instead, let's just output via console.log and let the user decide any additional piping they want to do.

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run `./dist/bin/vip-app.js 1513` and `./dist/bin/vip-app-list.js`
1. Both should have output rendered on-screen without being passed to less
2. Add `--format json` or `--format csv` and validate the same.
